azure-samples/pom.xml

```diff
@@ -54,31 +54,6 @@
             <artifactId>azure</artifactId>
```

Comment:
Sample depends on azure, hence it should not need the SDKs, since azure already have them all.